user-tool/tool/src/java/org/sakaiproject/user/tool/UsersAction.java

```diff
@@ -806,12 +806,15 @@ public void doSave(RunData data, Context context)
 
 		// commit the change
 		UserEdit edit = (UserEdit) state.getAttribute("user");
+		String valueEmail = (String)state.getAttribute("valueEmail");
+		String oldEmail = (String)state.getAttribute("oldEmail");
 		if (edit != null)
 		{
 			try
 			{
-				String valueEmail = (String)state.getAttribute("valueEmail");
-				if (StringUtils.trimToNull(valueEmail) != null && EmailValidator.getInstance().isValid(edit.getEid()) && !edit.getEid().equalsIgnoreCase(valueEmail)) {
+				//start this validation only when user/admin has changed the email for the account else skip
+				if (StringUtils.trimToNull(valueEmail) != null && StringUtils.trimToNull(oldEmail) != null && !(oldEmail.equals(valueEmail))
+						&& EmailValidator.getInstance().isValid(edit.getEid()) && !edit.getEid().equalsIgnoreCase(valueEmail)) {
 					validationLogic.createValidationAccount(edit.getId(),valueEmail);
 					addAlert(state,rb.getFormattedMessage("useedi.val.email",new String[]{valueEmail}));
```

Comment:
we still need to save the user object because there could be other changes like firstname,password,etc